tetris.js

```diff
@@ -447,8 +443,13 @@ module.exports = {
 				return this.coordinates
 			}
 		}
-		moveTetromino(){
-			
+		moveTetromino(field, shift){
+			let shiftTetrominoResult = module.exports.shiftCoordinates(this.coordinates, shift)
+			if(module.exports.checkThatTheFieldIsFree(shiftTetrominoResult, field)){
+				return shiftTetrominoResult
```

Comment:
this is not the full algorithm

Add tests that move 2 times in different directions, like down-left, right-down, left-right and check the result is as expected